tools/gcp_cli.py

```diff
@@ -285,3 +285,22 @@ def DeleteObject(args: 'argparse.Namespace') -> None:
   gcs.DeleteObject(args.path)
 
   print('Object deleted.')
+
+
+def InstanceNetworkQuarantine(args: 'argparse.Namespace') -> None:
+  """Put a Google Cloud instance in network quarantine.
+
+    Args:
+      args (argparse.Namespace): Arguments from ArgumentParser.
+  """
+  exempted_ips = []
+  if args.exempted_src_ips:
+    exempted_ips = args.exempted_src_ips.split(',')
+    # Check if exempted_src_ips argument exists and if there
+    # are any empty entries.
+    if not (exempted_ips and all(elements for elements in exempted_ips)):
```

Comment:
I *think* you can do `all(exempted_ips)`
```suggestion
    if not (exempted_ips and all(exempted_ips)):
```

Reply:
oh, that is indeed right